test/validators.js

```diff
@@ -8722,6 +8722,335 @@ describe('Validators', () => {
+          '+3547000000',
+          '+3547010000',
+          '+3547020000',
+          '+3547030000',
+          '+3547040000',
+          '+3547050000',
+          '+3547060000',
+          '+3547070000',
+          '+3547080000',
+          '+3547090000',
+          '+3547100000',
+          '+3547110000',
+          '+3547120000',
+          '+3547130000',
+          '+3547140000',
+          '+3547150000',
+          '+3547160000',
+          '+3547170000',
+          '+3547180000',
+          '+3547190000',
+          '+3547200000',
+          '+3547210000',
+          '+3547220000',
+          '+3547230000',
+          '+3547240000',
+          '+3547250000',
+          '+3547260000',
+          '+3547270000',
+          '+3547280000',
+          '+3547290000',
+          '+3547300000',
+          '+3547310000',
+          '+3547320000',
+          '+3547330000',
+          '+3547340000',
+          '+3547350000',
+          '+3547360000',
+          '+3547370000',
+          '+3547380000',
+          '+3547390000',
+          '+3547400000',
+          '+3547410000',
+          '+3547420000',
+          '+3547430000',
+          '+3547440000',
+          '+3547450000',
+          '+3547460000',
+          '+3547470000',
+          '+3547480000',
+          '+3547490000',
+          '+3547500000',
+          '+3547510000',
+          '+3547520000',
+          '+3547530000',
+          '+3547540000',
+          '+3547550000',
+          '+3547560000',
+          '+3547570000',
+          '+3547580000',
+          '+3547590000',
+          '+3547940000',
+          '+3547950000',
+          '+3547960000',
+          '+3547970000',
+          '+3547980000',
+          '+3547990000',
+          '+3548000000',
+          '+3548001000',
+          '+3548002000',
+          '+3548003000',
+          '+3548004000',
+          '+3548005000',
+          '+3548006000',
+          '+3548007000',
+          '+3548008000',
+          '+3548009000',
+          '+3548010000',
+          '+3548020000',
+          '+3548030000',
+          '+3548040000',
+          '+3548050000',
+          '+3548060000',
+          '+3548070000',
+          '+3548080000',
+          '+3548090000',
+          '+3548100000',
+          '+3548110000',
+          '+3548120000',
+          '+3548130000',
+          '+3548140000',
+          '+3548150000',
+          '+3548160000',
+          '+3548170000',
+          '+3548180000',
+          '+3548190000',
+          '+3548260000',
+          '+3548270000',
+          '+3548280000',
+          '+3548700000',
+          '+3548710000',
+          '+3548720000',
+          '+3548730000',
+          '+3548740000',
+          '+3548750000',
+          '+3548760000',
+          '+3548770000',
+          '+3548780000',
+          '+3548790000',
+          '+3548800000',
+          '+3548810000',
+          '+3548840000',
+          '+3548850000',
+          '+3548860000',
+          '+3548870000',
+          '+3548890000',
+
```

Comment:
I personally like the verbosity of tests here, but not sure what the maintainers of this project think though.

However, if you could get rid of the empty line here, that would be great :-)

Reply:
Removed it.